app/src/main/java/org/simple/clinic/search/results/PatientSearchResultsScreen.kt

```diff
@@ -114,20 +116,16 @@ class PatientSearchResultsScreen :
     context.injector<Injector>().inject(this)
   }
 
-  @SuppressLint("CheckResult")
-  override fun onFinishInflate() {
-    super.onFinishInflate()
-    if (isInEditMode) {
-      return
-    }
-
+  override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+    super.onViewCreated(view, savedInstanceState)
     setupScreen()
+    searchResultsView.searchWithCriteria(screenKey.criteria)
   }
 
   override fun onAttachedToWindow() {
     super.onAttachedToWindow()
     delegate.start()
-    searchResultsView.searchWithCriteria(screenKey.criteria)
+
```

Comment:
Can we remove this extra line please?

Reply:
This line was removed in a later commit.

Reply:
Oh sorry, missed it.